packs/data/pathfinder-bestiary-3.db/android-infiltrator.json

```diff
@@ -671,7 +671,42 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "nanite-surge",
```

Comment:
The skill bonus and the glow have a different duration, so to do it properly it would need to be an effect for the glow and the toggle for the bonus.

Reply:
Changed